keps/sig-network/4410-k8s-network-interface/README.md

```diff
@@ -40,21 +40,17 @@ and accommodate advanced functionalities and potential areas for expansion.
 ### Goals
 
 1. Design a cool looking t-shirt
-2. Design and implement the KNI-API
-3. Provide documentation, examples, troubleshooting and FAQ's for KNI.
```

Comment:
I think bullet point 3 is valuable as its a requirement for the KEP. Unless the documentation is an implicit requirement of the KEP?

Reply:
Sure, I think its _implicit_ anyhow, but let's make it explicit.